docs/config.md

```diff
@@ -197,6 +201,47 @@ names:
   commonNamePattern: "{host}.buoyant.io"
 ```
 
+#### Load Balancer
+
```

Comment:
it might be good to note here what default load balancer is used if this option is unspecified (unless I missed that?)